src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,563 @@
+_frame throughput_.
+
+If the GPU is really bogged down, the browser (or platform) may even begin to
+throttle the rate at which it attempts visual updates and thus decreases
+effective frame rates. While technically that can reduce the number of dropped
+frame updates, visually it will still appear as a lower frame throughput.
+
+Yet, not all types of low frame throughput are bad. If the page is mostly idle
+and there are no active animations, a low frame rate is just as visually
+appealing as a high frame rate (and, it can save battery!).
+
+So when does frame throughput matter?
+
+### Detecting animations
+
+High frame throughput matters especially during periods with important
+animations. Different animation types will depend on visual updates from a
+specific thread (main, compositor, or a worker), so its visual update is
+dependent on that thread providing its update within the deadline. We say that a
+given thread _affects smoothness_ whenever there is an active animation that
+depends on that thread update.
+
+Some types of animations are easier to define and detect than others.
+Declarative animations, or user-input-driven animations, are clearer to define
+than JavaScript-driven animations implemented as periodic updates to animatable
+style properties.
+
+Even with `requestAnimationFrame()`, which is designed for animations, you
+cannot always assume that every rAF call is necessarily producing a visual
+update or animation. For example, using rAF polling just to track frame rate
+(as shown above) should not itself affect smoothness measurements since there is
+no visual update.
+
+{% Aside %}
+  Animation detection in tooling, as well as the specifics of
+  animation implementations in browsers, continually evolves and improves.
+  Chrome recently moved [background-color
+  animations](https://developer.chrome.com/blog/hardware-accelerated-animations/#whats-coming-next)
+  from the main thread to the compositor!
+{% endAside %}
+
+### Quality versus quantity
+
+Finally, detecting animations and animation frame updates is still only part of
+the story because it only captures the quantity of animation updates, not the
+quality.
+
+For example, you may see a perfectly steady stream of 60&nbsp;fps while watching a
+video. Technically, this is perfectly smooth. But the video itself may have a
+low bit rate, or even issues with network buffering. This isn't captured by
+animation smoothness metrics directly, yet may still be quite jarring to the
+user.
+
+Or, a game which leverages `<canvas>` (perhaps even using techniques like
+[offscreen
+canvas](https://developers.google.com/web/updates/2018/08/offscreen-canvas) to
+ensure a steady frame rate) may technically be perfectly smooth in terms of
+animation frames, all while failing to load high quality game assets into the
+scene or exhibiting rendering artifacts.
+
+And of course, a site can just have some really lame animations :)
```

Comment:
Do plaintext emoticons get translated to emoji automatically? If not, let's just inline the standard smiley emoji in lieu of plaintext 🙂

Reply:
SGTM